app/timetables/models.py

```diff
@@ -39,3 +39,26 @@ def clean(self):
 
     def __str__(self):
         return self.name
+
+
+class Timetable(ForceCapitalizeMixin, TimestampMixin):
+    """
+    A timetable is the central entity or model of the platform. 
+    
+    It represents/encapsulates the entire structure and 
+    scheduling of meals, menu-items, dishes, courses, options etc, 
+    served at a location, to a team or the entire organisation.
+
+    Timetable model has a many-to-many relationship with the Weekday model.
+    """
+
+    name = models.CharField(max_length=60, unique=True)
+    code = models.CharField(max_length=60, unique=True)
+    api_key = models.CharField(max_length=60, unique=True)
```

Comment:
Another question is how unique is unique, "A" != "a"
Not forcing it would give you a wider range or alphanumeric characters to use that forcing a particular case. Unless making it all uppercase is the goal here.